src/api/cpp/cvc5_kind.h

```diff
@@ -88,8 +88,10 @@ enum CVC4_EXPORT Kind : int32_t
    *   - 1..n: Terms with same sorts
    *
    * Create with:
-   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2) const`
-   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2, const Term& child3) const`
+   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2)
+   * const`
+   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2,
+   * const Term& child3) const`
```

Comment:
Can you check that this does not break the link generation?

Reply:
Still seems to work:

![snap0087](https://user-images.githubusercontent.com/4653598/115457560-8ff36400-a1d9-11eb-9743-294ba58c946e.png)